example/cpp2/tests/fixeddecimal.cpp

```diff
@@ -18,11 +18,11 @@ int main(int argc, char *argv[]) {
 
     simple_assert_eq("Multiplying FixedDecimal", fd_out, "12.3");
 
-    std::string out;
+    // std::string out;
 
     // fd->to_string_to_writeable(out);
 
-    simple_assert_eq("Formatting FixedDecimal to Writeable", fd_out, "12.3");
+    // simple_assert_eq("Formatting FixedDecimal to Writeable", fd_out, "12.3");
```

Comment:
This is the test for the "append to writeable" rather than "return string" methods we had in C++1. I assume we probably still want these in C++2 but I can add those in a standalone PR.

Reply:
works for me!